RealmSwift/Impl/ComplexTypes.swift

```diff
@@ -209,8 +209,12 @@ extension RealmOptional: SchemaDiscoverable, _RealmSchemaDiscoverable where Valu
 }
 
 extension LinkingObjects: _Persistable where Element: _Persistable {
-    public static func _rlmDefaultValue() -> Self {
-        fatalError("LinkingObjects properties must set the origin property name")
+    public static func _rlmDefaultValue(_ doNotReturnNilValue: Bool) -> Self {
+        if doNotReturnNilValue {
+            return Self.init(propertyName: "", handle: nil)
```

Comment:
.init is redundant.

Reply:
This is required when `Persisted.get` is called.